climada/hazard/trop_cyclone/trop_cyclone.py

```diff
@@ -0,0 +1,696 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define TC wind hazard (TropCyclone class).
+"""
+
+__all__ = ['TropCyclone']
+
+import copy
+import datetime as dt
+import itertools
+import logging
+import time
+from typing import Optional, Tuple, List
+
+import numpy as np
+from scipy import sparse
+import matplotlib.animation as animation
+from tqdm import tqdm
+import pathos.pools
+import xarray as xr
+
+from climada.hazard.base import Hazard
+from climada.hazard.tc_tracks import TCTracks
+from climada.hazard.tc_clim_change import get_knutson_criterion, calc_scale_knutson
+from climada.hazard.centroids.centr import Centroids
+from climada.hazard.trop_cyclone.trop_cyclone_windfields import DEF_MAX_DIST_EYE_KM, DEF_INTENSITY_THRES, \
+    DEF_MAX_MEMORY_GB, compute_windfields_sparse
```

Comment:
line too long, why not use local import?
```suggestion
from .trop_cyclone_windfields import DEF_MAX_DIST_EYE_KM, DEF_INTENSITY_THRES, \
    DEF_MAX_MEMORY_GB, compute_windfields_sparse
```